src/index.html

```diff
@@ -1,17 +1,174 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
   </head>
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+
+      <fieldset class="block">
+        <legend>Personal information</legend>
+
+        <label for="Personal-information" class="form-field">
+          Surname:
+          <input id="Personal-information"
+            type="text"
+            name="surname"
+            class="field"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label for="Name" class="form-field">
+          Name:
+          <input id="Name"
+            type="text"
+            name="Name"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label for="Age" class="form-field">
+          How old are You?
+          <input id="Age"
+            type="number"
+            name="Age"
+            min="1"
+            max="100"
+            placeholder="12"
+            required
+          >
+        </label>
+
+        <label for="Date-of-Birth" class="form-field">
+          Full date of Birth:
+          <input id="Date-of-Birth"
+            type="date"
+            name="birth-date"
+            required
+          >
```

Comment:
fix indentation for every input

```suggestion
          <input
            id="Date-of-Birth"
            type="date"
            name="birth-date"
            required
          >
```